project/AppDependencies.scala

```diff
@@ -2,27 +2,27 @@ import sbt.*
 
 object AppDependencies {
 
-  private val bootstrapVersion = "9.18.0"
-  private val hmrcMongoVersion = "2.7.0"
+  private val bootstrapVersion = "9.19.0"
```

Comment:
How much work would it be to upgrade to 10.4.0? I see there were some breaking changes, so it might not be feasible, but it'd be good to get this up to date if possible.